### PR TITLE
Create community, events, knowledge pages

### PR DESCRIPTION
**Pull request by nathankota**
need to find unique images for each section and work on the action links

### CODE REVIEW

File: src/pages/collaboration/community.astro [hunk 1]
```diff
@@ -27,27 +52,66 @@ const page: IStandardPageMetadata = {
...
-      <li>Utah Valley GIS Users Group</li>
-      <li>G5 - Southwestern Utah counties</li>
-      <li>Uintah Basin User Group (UBUG)</li>
-      <li>Salt Lake User Group (SLUG)</li>
-    </ul>
+  <Section {...page.section[2]}>
+    <Image
+      slot="image"
+      src={communityPhoto}
+      loading="eager"
```
Comment (steveoh) [inline src/pages/collaboration/community.astro:71]: **quibble**: this property is only on the first image. you can remove it from every subsequent image

File: src/pages/collaboration/community.astro [hunk 2]
```diff
@@ -27,27 +52,66 @@ const page: IStandardPageMetadata = {
 <Layout title={page.title}>
   <QuickLinks links={page.section} slot="quick-links" />
   <Section {...page.section[0]}>
+  UGRC is honored to be a part of and participate in many user groups, councils, committees, associations, and communication channels. Join us in one or more of these groups to share and learn together through presentations, networking, and formal and casual communication. We look forward to meeting and working with you!
```
Comment (steveoh) [inline src/pages/collaboration/community.astro:55]: **quibble**: every paragraphs should be surrounded by `<p></p>` tags

**Review by steveoh — COMMENTED** [review]
looks good so far. i'm going to convert to a draft while you work on it.

File: src/pages/collaboration/community.astro [hunk 3]
```diff
@@ -8,17 +8,47 @@ import Layout from '@layouts/FullWidth.astro';
...
+      actionText: 'Learn more about UGIC',
+    },
+    {
+      title: 'National States Geographic Information Council (NSGIC)',
+      actionUrl: '/collaboration/somewhere',
+      actionText: 'Learn more about NSGIC',
+    },
+    {
+      title: 'Slack Spatial Community',
+      actionUrl: '/collaboration/somewhere',
```
Comment (steveoh) [inline src/pages/collaboration/community.astro:40]:
```suggestion
      actionUrl: 'https://thespatialcommunity.org/#join',
```

File: src/pages/collaboration/community.astro [hunk 4]
```diff
@@ -8,17 +8,47 @@ import Layout from '@layouts/FullWidth.astro';
...
+      actionText: 'Join the community',
+    },
+    {
+      title: 'Utah Association of Counties (UAC)',
+      actionUrl: '/collaboration/somewhere',
```
Comment (steveoh) [inline src/pages/collaboration/community.astro:45]:
```suggestion
      actionUrl: 'https://www.uacnet.org/',
```

File: src/pages/collaboration/community.astro [hunk 5]
```diff
@@ -8,17 +8,47 @@ import Layout from '@layouts/FullWidth.astro';
...
+      actionText: 'Learn more about UAC',
+    },
+    {
+      title: 'Utah Communications Authority (UCA)',
+      actionUrl: '/collaboration/somewhere',
```
Comment (steveoh) [inline src/pages/collaboration/community.astro:50]:
```suggestion
      actionUrl: 'https://www.uca911.org/',
```

File: src/pages/collaboration/community.astro [hunk 6]
```diff
@@ -8,17 +8,47 @@ import Layout from '@layouts/FullWidth.astro';
...
     {
-      title: 'Demonstration heading',
+      title: 'GIS user groups',
       actionUrl: '/collaboration/somewhere',
-      actionText: 'Link text... can be empty for default value',
+      actionText: 'Get involved',
+    },
+    {
+      title: 'Utah Geographic Information Council (UGIC)',
+      actionUrl: '/collaboration/somewhere',
```
Comment (steveoh) [inline src/pages/collaboration/community.astro:30]:
```suggestion
      actionUrl: 'https://ugic.org/',
```

File: src/pages/collaboration/community.astro [hunk 7]
```diff
@@ -8,17 +8,47 @@ import Layout from '@layouts/FullWidth.astro';
...
+      actionText: 'Learn more about UGIC',
+    },
+    {
+      title: 'National States Geographic Information Council (NSGIC)',
+      actionUrl: '/collaboration/somewhere',
+      actionText: 'Learn more about NSGIC',
+    },
+    {
+      title: 'Slack Spatial Community',
```
Comment (steveoh) [inline src/pages/collaboration/community.astro:39]:
```suggestion
      title: 'The Spatial Community',
```

File: src/pages/collaboration/community/user-groups.astro [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+const page: IStandardPageMetadata = {
+  title: 'Utah GIS user groups',
+  section: stripeSections([
+    { title: `Canyon Country Partnership Geographic Data Committee` },
+    { title: 'Central Utah User Group' },
+    { title: 'Northern Utah GIS' },
+    { 
+      title: 'Salt Lake User Group'
+      actionUrl: 'https://www.slug-gis.info/',
+      actionText: 'Visit SLUG's website',
```
Comment (steveoh) [inline src/pages/collaboration/community/user-groups.astro:19]:
```suggestion
      actionText: `Visit SLUG's website`,
```

File: src/pages/collaboration/community/user-groups.astro [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+import { stripeSections } from '@utils/page';
+
+const page: IStandardPageMetadata = {
+  title: 'Utah GIS user groups',
+  section: stripeSections([
+    { title: `Canyon Country Partnership Geographic Data Committee` },
+    { title: 'Central Utah User Group' },
+    { title: 'Northern Utah GIS' },
+    { 
+      title: 'Salt Lake User Group'
```
Comment (stdavis) [inline src/pages/collaboration/community/user-groups.astro:17]:
```suggestion
      title: 'Salt Lake User Group',
```

**Comment by nathankota** [comment]
thank you gentlemen 🙏

File: src/pages/collaboration/events.astro
Comment (nathankota) [inline src/pages/collaboration/events.astro:1]: thank you 🙏

**Pull request merged**